### DISCUSSION TITLE
Switching between perspective and orthogonal view for GUIVisualizer or SceneWidget+Camera

### DISCUSSION THREAD

**Discussion by hduonggithub** [isl-org/Open3D#3840] [Q&A]
Is there anyone know how to switch from perspective to orthogonal view (vice versa) keeping the same view size of model and zooming within GUIVisualizer or Scenewidget (vis_gui.py).

I read help/documentation a lot but still cannot figure out how to get it works. I wish I could have a example code/ hints how to do it. 

Help please ! Thanks

**Comment by germanros1987** [comment]
Please, check this discussion: https://github.com/isl-org/Open3D/discussions/3660
